environment_arm.yml

```diff
@@ -0,0 +1,78 @@
+channels:
+  - conda-forge
+  - usgs-astrogeology
+  
+dependencies:
+  - ale =0.10.0,<1
+  - aom
+  - armadillo
+  - boost >=1.76.0,<1.82
+  - boost-cpp >=1.76.0,<1.82
+  - blas
+  - usgs-astrogeology::bullet
+  - bz2file
+  - bzip2
+  - cmake >=3.27,<3.28
+  - cspice =67
+  - usgs-astrogeology::csm >=3.0.3,<3.0.4
+  - curl >=8.4.0,<8.5
+  - cxx-compiler=1.7
+  - doxygen
+  - eigen >=3, <4
+  - embree >=3, <4
+  - ffmpeg
+  - geos >=3.12,<3.13
+  - geotiff
+  - gdal >=3.7,<3.9
+  - gmp
+  - graphviz
+  - conda-forge::gsl >=2.6, <2.7
+  - hdf5
+  - highfive
+  - icu
+  - usgs-astrogeology::inja
+  - jama
+  - kalasiris
+  - krb5
+  - libgdal >=3.7,<3.9
+  - libopencv >=4.5,<4.10
+  - libpng >=1.6.34,<1.7
+  - libprotobuf
+  - libtiff >=4.5,<4.8
+  - libxml2
+  - make
+  - mesalib
+  - mysql
+  - nanoflann <1.4
+  - nlohmann_json=3.11.2
+  - ninja >=1.10,<1.11
+  - nn
+  - opencv >=4.5,<4.10
+  - openssl >=3.1.2,<3.5
+  - pcl >=1.13,<1.14
+  - pdal=2.6
+  - plio
+  - protobuf
+  - python >=3.9
+  - python_abi >=3.9
+  - pytest
+  - rclone
+  - qhull
+  - qt-main>=5.15, <5.16
+  - qwt <6.3.0
+  - sqlite >=3.46.0,<3.47
+  - suitesparse <7.7.0
+  - superlu
+  - swig
+  - texlive-core
+  - tnt
+  - usgs-astrogeology::usgscsm
```

Comment:
This is on conda-forge not the usgs-astrogeology channel

Reply:
Switch it to conda-forge and getting the following dep error:
```
Could not solve for environment specs
The following packages are incompatible
├─ usgscsm is installable and it requires
│  └─ python_abi 3.12.* *_cp312, which can be installed;
└─ xalan-c is not installable because it requires
   └─ python_abi 3.10.* *_cp310, which conflicts with any installable versions previously reported.
```

Reply:
usgscsm has been updated on conda-forge so this just needs the `usgs-astrogeology::` removed.